ci-operator/config/shiftstack/merge-bot/shiftstack-merge-bot-main.yaml

```diff
@@ -213,15 +214,15 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
-- as: cloud-provider-openstack-4-18
+- as: cloud-provider-openstack-main
   cron: 0 12 * * Mon,Thu
   steps:
     test:
-    - as: cloud-provider-openstack-4-18
+    - as: cloud-provider-openstack-main
       commands: |
-        merge-bot --source https://github.com/kubernetes/cloud-provider-openstack:release-1.31 \
-                  --dest openshift/cloud-provider-openstack:release-4.18 \
-                  --merge shiftstack/cloud-provider-openstack:merge-bot-release-4.18 \
+        merge-bot --source https://github.com/kubernetes/cloud-provider-openstack:release-1.32 \
```

Comment:
That's fine, but we'll have to be careful to first merge a rebase on 1.32 before merging anything that merge-bot proposes to CAPO main. I suspect it won't propose anything anyway due to conflicts until we rebase main.